src/NuGet.Clients/NuGet.CommandLine/Commands/Command.cs

```diff
@@ -211,7 +211,7 @@ protected void SetDefaultCredentialProvider(Lazy<string> msbuildDirectory)
             var pluginProviders = new PluginCredentialProviderBuilder(extensionLocator, Settings, Console)
                 .BuildAll(Verbosity.ToString())
                 .ToList();
-            var securePluginProviders =  await (new SecureCredentialProviderBuilder(PluginManager.Instance, Console)).BuildAll();
+            var securePluginProviders =  await (new SecurePluginCredentialProviderBuilder(pluginManager: PluginManager.Instance, canShowDialog: true, logger: Console)).BuildAll();
```

Comment:
if BuildAll is an async call, why is the name not BuildAllAsync?

Reply:
I think it was synchronous at some point and made async later. 
Nice catch, i will chnage that.